tests/unit-tests/ambiguous/__snapshots__/index.test.ts.snap

```diff
@@ -1938,17 +1916,16 @@ export interface Signature {
   Yields: [];
 }
 
-<template>
+export default (<template>
```

Comment:
Looks like we picked up an accidental `export default` here.

Reply:
its because i'm leaving  `as TemplateOnlyComponent<Signature>` to prettiers default handling.
Since it now remains an expression, we cannot safely remove or add default exports to it.
are you okay with this?

Reply:
Seems fine.